indicator-nara/035-berekening.Rmd

```diff
@@ -0,0 +1,879 @@
+
+Figuur \@ref(fig:droogte-indicator-met-ci-figuur) is hiervan een grafische weergave.
+De groene lijn in de figuur geeft het aantal dagen dat jaarlijks gemiddeld verwacht wordt.
+
+Het grillige patroon van de indicator springt hierbij in het oog.
+Een 'normaal' jaar is eerder uitzondering dan regel.
+Als het droog is, is het ook vaak flink droog. 
+Natte jaren zijn minder opvallend, omdat de droogte-indicator niet onder nul kan gaan.
+Uitgezonderd voor het jaar 2018 zit er vrij weinig variatie in de hoogte van de pieken. 
+Men ziet wel een wijziging in de frequentie ervan, met een toename vanaf rond 2000.
+Dit wordt verder onderzocht in het onderzoek van de trend.
+
+(ref:droogte-indicator-met-ci-figuur) Droogte-indicator met 95% betrouwbaarheid
+
+```{r droogte-indicator-met-ci-figuur, fig.cap= "(ref:droogte-indicator-met-ci-figuur)"}
+g <- ggplot(data = droogte_ind_j, aes(x = jaar, y = droogte_ind_j)) + 
+  theme_inbo() +
+  geom_ribbon(aes(ymin = lcl, ymax = ucl)) +
+  geom_hline(yintercept = 365*0.05, color = "green", linetype = "dashed") +
+  geom_line(color = "red", size = 0.5) + 
+  labs (y = "aantal droge dagen") + 
+  scale_x_continuous(n.breaks = nrow(droogte_ind_j)) + 
+  theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))
+g
+```
+
+De formule van Rubin laat toe om in te schatten wat de kost is van het ontbreken van data op de betrouwbaarheid van het resultaat.
+Onderstaande figuur (\@ref(fig:droogte-indicator-imput-fig)) toont het aandeel in de totale variatie ten gevolge van het schatten van ontbrekende gegevens. 
+Dit is meestal meer dan 10% en kan tot 50% bedragen.
+Er zit een duidelijke neerwaartse trend in.
+Deze trend is te verklaren door de toegenomen meetintensiteit.
+
+```{r droogte-indicator-imput-fig, fig.cap="Het aandeel van imputatie in de totale variatie van de droogte-indicator", message=FALSE}
```

Comment:
I don't understand this figure.

Reply:
I have changed the label of the y-axis: "aandeel variatie door imputatie in de totale variatie (%)". Is it more clear ?
I want to show that in the beginning much more data needed to be imputated and i want to show the cost of it. 
Also, with this figure one can estimate what the benefit is of a good datacollection-policy (no imputations). I'm a right ?